go/cmd/yolo/main.go

```diff
@@ -89,6 +90,7 @@ func yolo(args []string) error {
 	serverFlagSet.StringVar(&bintrayToken, "bintray-token", "", "Bintray API Token")
 	serverFlagSet.StringVar(&circleciToken, "circleci-token", "", "CircleCI API Token")
 	serverFlagSet.StringVar(&githubToken, "github-token", "", "GitHub API Token")
+	serverFlagSet.StringVar(&githubRepos, "github-repos", "", "GitHub repositories to watch")
```

Comment:
```suggestion
	serverFlagSet.StringVar(&githubRepos, "github-repos", "berty/berty", "GitHub repositories to watch")
```

better with a default-value :)

Reply:
(still for the precious `--help` feature)